lib/ops/version.rb

```diff
@@ -1,3 +1,3 @@
 module Ops
-  VERSION = '0.5.2'
+  VERSION = '0.5.3'
```

Comment:
i see. i was thinking of it as a bug fix because the links are busted. you could argue its a feature as well. i'll update the version